ui/src/mixins/MobileLayout.js

```diff
@@ -0,0 +1,209 @@
+            switch (screenSize) {
+              case 'xs': style='xs8 title text-xs-left';
+                break;
+              case 'sm': style='xs8 title text-xs-left';
+                break;
+              case 'md': style='xs6 display-2 text-xs-left';
+                break;
+              case 'lg': style='xs6 display-2 text-xs-left';
+                break;
+              case 'xl': style='xs6 display-2 text-xs-left';
+                break;
+            }
+            return style;
+        },
+        cardTitleStyle(screenSize){
+            var style="";
+            switch (screenSize) {
+              case 'xs': style='title';
+                break;
+              case 'sm': style='title';
+                break;
+              case 'md': style='display-1';
+                break;
+              case 'lg': style='display-1';
+                break;
+              case 'xl': style='display-1';
+                break;
+            }
+            return style;
+        },
+        fontSizeParagraph(screenSize){
+            var fontSize='';
+            switch (screenSize) {
+              case 'xs': fontSize = '12px'
+                break;
+              case 'sm': fontSize =  '12px'
+                break;
+              case 'md': fontSize =  '14px'
+                break;
+              case 'lg': fontSize =  '16px'
+                break;
+              case 'xl': fontSize =  '16px'
+                break;
+            }
+            return fontSize;
+        },
+        gridLayoutShowRecord(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12 pr-3 ml-2';
+                break;
+              case 'sm': layout = 'xs8 pr-3 ml-2';
+                break;
+              case 'md': layout = 'xs6 pr-3';
+                break;
+              case 'lg': layout = 'xs6 pr-3';
+                break;
+              case 'xl': layout = 'xs6 pr-3'
+            }
+            return layout;
+        },
+        gridLayoutSearchResult(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12';
+                break;
+              case 'sm': layout = 'xs9';
+                break;
+              case 'md': layout = 'xs9';
+                break;
+              case 'lg': layout = 'xs9';
+                break;
+              case 'xl': layout = 'xs9'
+            }
+            return layout;
+        },
+        gridLayoutShowResult(screenSize){
+            var layout = "";
+            switch (screenSize) {
+              case 'xs': layout = 'xs12';
+                break;
+              case 'sm': layout = 'xs12';
+                break;
+              case 'md': layout = 'xs9';
+                break;
+              case 'lg': layout = 'xs9';
+                break;
+              case 'xl': layout = 'xs9'
+            }
+            return layout;
+        },
+    smallScreenCompute(screenSize){
```

Comment:
These methods seem similar to the computed properties. Do you need both? Also, it's not clear from the method name what this does.